README.md

```diff
@@ -1,3 +1,11 @@
+# This repository is no longer maintained. See [Jellyfish Videoroom] instead.
+
+Because of a lot of projects repeating `Membrane Videoroom`'s logic, we extracted it into general purpose media server called [Jellyfish].
+While this repository is still a great example on how to use [Membrane RTC Engine] to
```

Comment:
If we don't maintain this, it won't be a 'great example of how to use [Membrane RTC Engine]' in a month or two